dashboards/lambdatest/lambdatest-logs.json

```diff
@@ -0,0 +1,543 @@
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": true
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [
+                  4183939
+                ],
+                "query": "SELECT count(*) FROM Log\nWHERE `service-name`='lambdatest' \nAND `newrelic.source` = 'api.logs' \nAND error_message IS NOT NULL\nAND `error_message` != '-'\nFACET error_message\nLIMIT 10\nSINCE 1 week ago"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Test Status Ratio",
+          "layout": {
+            "column": 1,
+            "row": 9,
+            "width": 12,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.stacked-bar"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [
+                  4183939
+                ],
+                "query": "SELECT \n  FILTER(count(*), WHERE status IN ('Lambda Error', 'Queue_Timeout', 'Idle_Timeout', 'Failed')) AS 'Failed',\n  FILTER(count(*), WHERE status NOT IN ('Lambda Error', 'Queue_Timeout', 'Idle_Timeout', 'Failed')) AS 'Success'\nFROM Log\nWHERE `service-name`='lambdatest' \nAND `newrelic.source` = 'api.logs' \nAND test_id IS NOT NULL \nAND status IS NOT NULL \nSINCE 1 week ago TIMESERIES 1 hour"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Test Failure by Name",
+          "layout": {
+            "column": 1,
+            "row": 12,
+            "width": 12,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.stacked-bar"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [
+                  4183939
+                ],
+                "query": "SELECT count(*) \nFROM Log \nWHERE `service-name`='lambdatest' \nAND `newrelic.source` = 'api.logs' \nAND status IN ('Idle_Timeout', 'Queue_Timeout', 'Lambda Error')\nFACET test_name, hourOf(timestamp)\nSINCE 1 week ago TIMESERIES 1 hour\n"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": []
+}
```

Comment:
The dashboard sanitization script needs to be ran for this. You can follow the steps in the [contributing guide](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) to run it. Other than that it looks great!

Reply:
Updated dashboard.